editor-components/stats/stats7/stats7.tsx

```diff
@@ -2,103 +2,119 @@ import * as React from "react";
 import { BaseStats } from "../../EditorComponent";
 import styles from "./stats7.module.scss";
 
-type ICard = {
-  subtitle: string;
+
+interface Item {
```

Comment:
```suggestion
type Item {
 title: JSX.Element;
 progress: number;
 progressText: JSX.Element;
}
```

[please check](https://github.com/Teknodev/composer-tools/blob/main/editor-components/README.md)